mllib/src/main/scala/org/apache/spark/ml/feature/Word2Vec.scala

```diff
@@ -49,6 +49,17 @@ private[feature] trait Word2VecBase extends Params
   def getVectorSize: Int = $(vectorSize)
 
   /**
+   * The window size (context words from [-window, window])
```

Comment:
Unrelated to this PR, but we should also have the defaults documented in the scaladocs